src/frontend/js/hooks/useAddresses.ts

```diff
@@ -38,6 +42,8 @@ export const useAddresses = () => {
 
   return {
     items: readHandler.data || [],
+    error: readHandler.error,
+    readHandler,
```

Comment:
Not a fan to pass the whole `readHandler` and also declare `items` and `error` properties which are parts of `readHandler`

Reply:
I can remove `readHandler` from the object, WDYT ?